### ISSUE TITLE
[BUGS] Developer tools

### ISSUE THREAD

**Issue by kryptobolt200528** [kiwibrowser/src.next#61]
The developers tools is not optimised for editing the code of a website ...and even making a mere change such as trying to remove a part of the code is hard .....
Btw,thank you dev for listening to our requests of including teh dev tools....

**Comment by kiwibrowser** [comment]
I agree, it needs some improvements just for info, there is a 3 dots menu button on the left of lines to help with the lack of right-click on mobile

**Comment by Lartsch** [comment]
Yep, it needs adjustments. But thanks for including it at all :)
What is especially bugging me out about it is that I switched to the Console view and am now stuck there. Guess I completely need to wipe Kiwi's data in order to get the default view back

**Comment by kiwibrowser** [comment]
It's fixed @Lartsch for the stuck in Console :)

**Comment by mirko77** [comment]
It does not even open, what am I missing? I see a sort of animation going to the bottom, that's it.

**Comment by evanbattaglia** [comment]
@mirko77 I thought this too, I finally figured out it was opening in a new tab :laughing: :facepalm:

**Comment by allytiago** [comment]
I see that there is now a DevTools similar to the desktop one, but a bit limited in the selector
